### PR TITLE
Gabriel/signout button

### PR DESCRIPTION
**Pull request by gdumouli**
- Added an "Unsign" button in Event_info activity 
I'm open to ideas if anyone has something better than "Unsign" for the name

### CODE REVIEW

File: code/noRAM/app/src/main/res/values/strings.xml
Comment (ColeDewis) [inline code/noRAM/app/src/main/res/values/strings.xml:97]: how about "Undo Signup" ? that might be a bit better?

**Review by ColeDewis — COMMENTED** [review]
Overall, this looks good - however does this have the changes from your other PR in it as well? it looks like it does; if that's the case we should wait to merge this until that one is done. I haven't approved simply for that reason - let me know, if its unrelated then ill approve, otherwise ill approve on the condition it waits for the other one. (Mainly doing this just to make sure you see it before merging)

**Comment by gdumouli** [comment]
> Overall, this looks good - however does this have the changes from your other PR in it as well? it looks like it does; if that's the case we should wait to merge this until that one is done. I haven't approved simply for that reason - let me know, if its unrelated then ill approve, otherwise ill approve on the condition it waits for the other one. (Mainly doing this just to make sure you see it before merging)

Ah good catch, I forgot to switch to master before creating this branch. I'll wait for the other PR to be merged before merging this one.

**Review by ColeDewis — APPROVED** [review]

**Pull request merged**
